### ISSUE TITLE
Downloading content 0%

### ISSUE THREAD

**Issue by pragat12** [ValveSoftware/Dota2-Gameplay#18725]
### Description

This has happened to me for 3rd time. After I select hero my game crashes. I try to open the game but it won't because downloading content is at 0%. I have 3 abandoned games which sucks. Is there any way to get the abandon reversed or fix the issues caused by this bug? Losing mmr and getting lp, kinda spoiling the whole thing. And you are correct its happening every few games in this patch. Never experienced it before.

### Example Match ID (and possibly Timestamp)

7757569929

**Comment by UnpopularOpinion** [comment]
facing same issue

**Comment by pragat12** [comment]
Have you fixed it yet?

**Comment by UnpopularOpinion** [comment]
This worked but it did:
1. restarted steam
2. verify game files
3. wait longer than usual but eventually it changes from 0%
4. press play game from steam library and it didn't download anything

Reading github and reddit, it seems that there's more than a few people
that experienced this issue after the 7.36 update. *The steps listed above
solved the loading back to the game but doesn't guarantee the crash doesn't
repeat.* The crash happens after selecting a hero, then stuck at the issue
we discussed here. Due to verifying / stuck downloading, most people face
abandonment. Some suggest starting the game from the steam library to avoid
this issue. Some suggest verifying before starting the game. Some suggest
watching a game through the watch tab before searching the queue.

**Comment by JosephSchafer** [comment]
Happening for me as well.  I got an error related to parsing a file or something about the battlepass 2016.  Not pleasant to have reports, and a timeout due to crashing.

**Comment by MisforMaTe** [comment]
+1, it crashes every single fucking game and refuses to open until the game is over. I got 3 timeouts already. Tried everything, reinstalling the game, steam, nothing.

**Comment by JosephSchafer** [comment]
Not sure if you are running an i9 but I changed a few bios settings and
that may have helped.

I’m not sure if it’s related - I did the bios change due to other BSODs
happening.  Dota has been stable since.

Other games like Palworld still causing me grief…

**Comment by dbybanez** [comment]
Happened to me after the Ringmaster update. Can any dev please take a look at this?

**Comment by UnpopularOpinion** [comment]
At one point this stopped happening to me. After ringmaster update it reemerged. Crazy that months later it is still happening.

**Comment by UnpopularOpinion** [comment]
relevant reddit posts:
https://www.reddit.com/r/DotA2/comments/1dvf43b/verify_integrity_of_game_files_error/
https://www.reddit.com/r/DotA2/comments/1dps2tn/downloading_content_0_validating_integrity_of/
https://www.reddit.com/r/DotA2/comments/1ddcq02/dota_2_verify_integrity_error_keeps_erroring/

**Comment by qherreros** [comment]
Seeing this error too in steam Flatpak with dota

**Comment by JosephSchafer** [comment]
This ended up being an issue with my Intel I9 14900K melting down due to
the now well-documented voltage issue.  RMA’d the whole computer.

**Comment by qherreros** [comment]
I'm seeing this only with DOTA, always after a strange corruption crash. It doesn't happen on other games (Last of Us / Neva / Warhammer / ...). And I don't have an intel processor. But given the number of graphical bug I see appear in DOTA months after months, I'm not sure it's solvable.

**Comment by t1gu1** [comment]
I got the same problem with 2 other game recently.

- Path of Exile 2
- Marvel Rivals

So it seems that it could happen to any games.

I try this:
- Verify the integrity of the game
- Clear steam cache
- Uninstall the game and reinstall it
- Try to change the region of download
- Wait a long time when it is stuck at `Downloading content (0%)`.

Nothing works for me sadly.

**Comment by ricardoreais** [comment]
Same thing happening to me but only on dota 2, other games are completely fine

**Comment by SpectrAlbert** [comment]
Still happening to me — this is the third or fourth time.
The worst part is, you don’t know what to do.
It happens randomly every few games.

Consequences: waiting 1 hour to play again, getting placed into the low-priority pool, and needing to win 2 games.

In my case, it started happening when, for some reason, I decided to try some INTERFACE announcer packs/mega-kill packs I had owned a while ago, replacing the default ones.

Then the game started crashing at the beginning of matches.
But restoring the default ones again hasn’t fixed it.

Frustrating...

**Comment by vvjacobo** [comment]
Happened to me today and previous week, always at start of a match and dumped me twice into low priority. Fix this asap, its a critical bug that I only solved by reinstaling dota again

**Comment by StupydHors** [comment]
Happening to me too.
Just got banned & low prio & lost behavior score because of this.
Game just instant crashes and you cannot get back in because of "Downloading content (0%)".

What I found helped is this.
1. Clear download cache (this restarts steam)
2. Log into a different account and launch dota (this will verify the files properly without getting stuck on "dowloading content" - it takes about 10 minutes on a pretty high end PC)
3. turn off dota
4. Switch back to main account
5. Enjoy playing from behind I guess

**Comment by Maravedis** [comment]
Encountering this problem on kernel `7.0.8-arch1-1` too, with hyprland as the WM. The above tip of logging out, using another account and logging out again worked. Would prefer not to be kicked out of game however.

**Comment by GlebZabl** [comment]
faced same issue about once per week, only changing steam account after pc restart works, every time got low priority and ban, does anyone found solution what fixed it faster then 5 minutes to avoid bans?
